packages/static-build/src/utils/build-output-v3.ts

```diff
@@ -0,0 +1,23 @@
+import { join } from 'path';
+import { promises as fs } from 'fs';
+
+export const BUILD_OUTPUT_DIR = '.vercel/output';
+
+/**
+ * Returns the path to the Build Output v3 directory when the
+ * `config.json` file was created by the framework / build script,
+ * or `undefined` if the framework did not create the v3 output.
+ */
+export async function getBuildOutputDirectory(
+  path: string
+): Promise<string | undefined> {
+  try {
+    const outputDir = join(path, BUILD_OUTPUT_DIR);
+    const configPath = join(outputDir, 'config.json');
+    await fs.stat(configPath);
```

Comment:
It's stat'ing a file (`config.json`). The reason it doesn't check for the directory itself is because in `vc build` the skeleton of the directory has already been created before the Builders are run, but the `config.json` doesn't get created until the end, so it's a better thing to check.